gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/listener/http/ListenerHttp.java

```diff
@@ -49,6 +49,7 @@
 public class ListenerHttp extends Listener {
```

Comment:
ListenerHttp or HttpListener ?

Reply:
In order to get the class in the proper order I prefer to keep the prefix with the implemented interface so when you search for Listener you have this : 
![image](https://user-images.githubusercontent.com/4324887/183036442-9b13dafa-7091-4a61-9e90-d4f33e4fed1c.png)

Reply:
We finally decide to align with classes where we chose to prefix. Will do that in another PR.